gravitee-apim-integration-tests/src/test/resources/proto/hello_streaming.proto

```diff
@@ -1,4 +1,5 @@
-// Copyright 2015 The gRPC Authors
+//
+// Copyright © 2015 The Gravitee team (http://gravitee.io)
```

Comment:
I don't think we should update this copyright...
I know the change is automatically applied with the `mvn license:format` command. We need to add an exclusion on that 🙈